pages/account/index.tsx

```diff
@@ -32,7 +56,16 @@ export async function getServerSideProps({ req }) {
     }
   }
 
-  return {
-    props: { isAuthenticated },
+  const id = patronTokenResponse.decodedPatron.sub
```

Comment:
Looks like there's a case where decoded patron can be null, which me think optional chaining should be used here. In addition, we should check for the existence of id before calling MyAccountFactory.